src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public User getDefaultValidUser() {
```

Comment:
why public?
this logic should also be called in the `beforeEach` method to reinitialize the `validUser` calss-field(which we also need to add) to not create this user every time in the tests themselves, but to just update a single field that we need it to have in the corresponding test.
this beforeEach should also clear the storage to have our tests isolated and not mix-up the test data between them